lib/screen/onboarding_page.dart

```diff
@@ -69,27 +69,28 @@ class _OnboardingPageState extends State<OnboardingPage>
     // if something goes wrong, we will catch it in the try catch block,
     // those issue can be ignored, let user continue to use the app
     try {
-      await DeviceInfo.instance.init();
-      await injector<DeviceInfoService>().init();
-      final metricClient = injector.get<MetricClientService>();
-      await metricClient.initService();
-      await injector<RemoteConfigService>().loadConfigs();
+      unawaited(DeviceInfo.instance.init());
```

Comment:
this task quick, doesn't need unawaited

Reply:
but i think we don't need to wait for anything unless it's necessary.